src/ansys/aedt/core/cli.py

```diff
@@ -331,5 +334,147 @@ def stop(
     return
 
 
+@panels_app.command("add")
+def add_panels(
+    aedt_version: str = typer.Option(
+        None,
+        "--version",
+        "-v",
+        help="AEDT version (e.g. 2025.2)",
```

Comment:
```suggestion
        help="AEDT version (such as 2025.2)",
```
Goggle dev style guide says to avoid Latin phrases like e.g. and i.e.